operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/Controller.java

```diff
@@ -228,20 +225,7 @@ public void start() throws OperatorException {
   }
 
   private ConfigurationService configurationService() {
```

Comment:
Do we need this method? Would just delete it.

Reply:
Done.